src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +26,29 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+
+        ArrayList<Product> products = new ArrayList<>();
+
+        Product product1 = new Product("Cooking Book", 10, BOOKS);
+        products.add(product1);
```

Comment:
creating and adding the product could be inlined. that way you would not need to create 6 variables that are only used once